src/main/java/core/basesyntax/service/activitieswithfruitservice/impl/BalanceFruitActivitiesServiceImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.activitieswithfruitservice.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activitieswithfruitservice.ActivitiesWithFruitService;
+
+public class BalanceFruitActivitiesServiceImpl implements ActivitiesWithFruitService {
+    @Override
+    public void activitiesWithFruit(FruitTransaction fruitTransaction) {
```

Comment:
Methods should be named with verbs to be more understandable
```suggestion
    public void performTransaction(FruitTransaction fruitTransaction) {
```

Reply:
++